recipes/mongodb_org_repo.rb

```diff
@@ -47,7 +47,7 @@
     keyserver 'hkp://keyserver.ubuntu.com:80'
     key package_version_major >= 3.2 ? 'EA312927' : '7F0CEB10'
   end
-when 'amazon', 'fedora', 'oracle', 'rhel'
+when 'rhel'
```

Comment:
Ah missed that, remove only `oracle` then :)
`amazon` and `fedora` are families